src/basis/basis.js

```diff
@@ -396,14 +397,23 @@ export class Basis {
      */
     get maxPairwiseDistance() {
         let maxDistance = 0;
-        for (let i = 0; i < this._elements.array.length; i++) {
-            for (let j = i + 1; j < this._elements.array.length; j++) {
-                const distance = math.vDist(
-                    this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
-                if (distance > maxDistance) {
-                    maxDistance = distance;
+        if (this._elements.array.length >= 2) {
+            for (let i = 0; i < this._elements.array.length; i++) {
+                for (let j = i + 1; j < this._elements.array.length; j++) {
+                    const distance = math.vDist(
+                        this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
+                    if (distance > maxDistance) {
+                        maxDistance = distance;
+                    }
                 }
             }
+        } else if(this._elements.array.length === 1) {
+            const element = this._elements.getArrayElementByIndex(0);
+            Object.keys(PERIODIC_TABLE).forEach(key => {
+                if (element === PERIODIC_TABLE[key].symbol) {
+                    maxDistance = math.precise(PERIODIC_TABLE[key].atomic_radius_pm * 0.01, 4);
+                }
+            });
```

Comment:
(1) I suggest moving the lines 411-416 to a separate function on documenting exactly what's happening therein, especially the "magic" numbers of 0.01 and 4. This new function has to be tested also.

(2) For the case when we have an atom that does not exist in the periodic table (say, "Si1" the if condition won't ever be satisfied

(3) The Foreach is a very inefficient implementation, we could just use `find`